pkg/sdkclient/sideinput/client.go

```diff
@@ -21,8 +21,6 @@ type client struct {
 	grpcClt sideinputpb.SideInputClient
 }
 
-var _ Client = (*client)(nil)
```

Comment:
Please keep this line. https://github.com/KeranYang/numaflow-notes/blob/main/GoFundamental.md#2-a-trick-to-ensure-an-object-implements-an-interface